pkg/app/app.go

```diff
@@ -164,9 +165,37 @@ func (a *App) Start() {
 		a.log.Notice("Broker successfully bootstrapped on startup")
 	}
 
+	interval, err := time.ParseDuration(a.config.Broker.RefreshInterval)
+	a.log.Debug("RefreshInterval: %v", interval.String())
+	if err != nil {
+		a.log.Error(err.Error())
+		a.log.Error("Not using a refresh interval")
+	} else {
+		ticker := time.NewTicker(interval)
+		ctx, cancelFunc := context.WithCancel(context.Background())
+		defer cancelFunc()
+		go func() {
+			for {
+				select {
+				case <-ticker.C:
+					a.log.Info("Broker configured to refresh specs every %v seconds", interval)
+					v := <-ticker.C
```

Comment:
I think that you want to do this in the case statement. 
```go
case v := <-ticker.C:
```
This will just block until the tick, I think at least am I wrong?